CONTRIBUTING.md

```diff
@@ -30,9 +30,6 @@ There are a few things that might look non-standard to most Lua programmers,
 which come about as `l3build`'s focus is testing and building LaTeX packages:
 
 - Our target Lua set up is `texlua` (part of LuaTeX), not standalone `lua`
-- The main `l3build.lua` file is self-contained as this helps with
-  bootstrapping LaTeX: we are aiming to maintain a single file with no
-  `.lua` dependencies
```

Comment:
Other than perhaps removing `a single file`  I think I'd keep this.

Reply:
I can restore this paragraph/item with more adaption, since `l3build.lua` is not self-contained any more (as well).

Reply:
As far as I know `l3build` has no dependencies beyond a TeX  system with LuaTeX?

Reply:
`l3build` is self-contained, but the file `l3build.lua` is not. I'll reword to use phrase `l3build` instead of `l3build.lua`.